recipes/salad/stir-fry-cabbage.md

```diff
@@ -0,0 +1,58 @@
+# Stir-Fry Cabbage
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-29%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-0.5hr-red.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-VeryCheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+* **First cooked by:** [@heathercoraje](https://github.com/Heathercoraje)
+* **Date:** Week 11, 13 FACN2
+* **Author of Recipe**: [@heathercoraje](https://github.com/Heathercoraje)
+
+**The story behind this dish**
+The author of recipe often suffers from acid reflux and indigestion. Cabbage is known to help reduce stomach pain and intestinal Ulcers.
+[Read more](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1643665/). Because fresh cabbage juice isn't her favorite beverage, she came up with a brilliant idea to eat a large amount of cabbage and actually enjoy eating it.
+
+```
+return Stir-Fry Cabbage
+```
+
+## Ingredients
+
+A wok, vegetable oil (olive oil x), 1/2 of a large white cabbage, 3 pieces of 50g of slim Scallion (Green onion), 2 pieces of large dried chili pepper, a base sauce, 5 or more cloves of garlic.
```

Comment:
I think these would be easier to read as a bullet point list :+1:

Reply:
but glasses